### PR TITLE
Hero

### LINKED ISSUE #4: Hero
- [x] background animation --> (css? js?)
- [x] I Am A DEVELOPER animation --> (css? js?)
- [x] social links --> js
- [x] Download CV button --> js
- [x] "Go down" button --> js

### PR DESCRIPTION
**Pull request by KarolisLuko**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    font-size: 20px;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    margin: 8px;
+    padding: 0px;
+    border-radius: 5px;
+}
+.Elivishs {
+    filter: invert(100%);
+}
+p{
```
Comment (GedasPupa) [inline css/main.css:39]: please, change common  selectors to class names / ids.

**Review by GedasPupa — CHANGES_REQUESTED** [review]
Please change selectors by tag name to class name. More info in discord

File: css/main.css (hunk 1 above)
Comment (vadimsharanov) [reply css/main.css:39]: Sutinku su Gediminu

**Comment by GedasPupa** [comment]
jau perkelta i developa/masteri

**Pull request closed without merging**